pfioh/pfioh.py

```diff
@@ -635,13 +638,13 @@ def ls_process(self, **kwargs):
 
         return d_ret
     
-    def getHeaders(self):
+    def getContentLength(self):
         """
         Return headers of the request
         """
         
         self.dp.qprint('headers= %s' %self.headers)
-        return self.headers['content-length']     
+        return int(self.headers['content-length'])
```

Comment:
This always ended up being cast to an int anyway, so I just moved that inside getContentLength